package.json

```diff
@@ -71,12 +71,11 @@
     "angular-in-memory-web-api": "0.1.13",
     "bootstrap": "^3.3.7",
     "core-js": "^2.4.1",
-    "dragula": "3.7.2",
     "lodash": "4.17.2",
     "moment": "2.17.1",
     "ng2-bootstrap": "1.1.16",
     "ng2-completer": "0.3.1",
-    "ng2-dragula": "1.2.1",
+    "ng2-dnd": "2.0.1",
```

Comment:
+1. On the UI the drag and drop experience is smooth especially dragging and scrolling.